core/src/main/java/tc/oc/pgm/api/event/FriendStatusChangeEvent.java

```diff
@@ -0,0 +1,30 @@
+package tc.oc.pgm.api.event;
+
+import java.util.UUID;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+/** Called when friendship status is updated for a player * */
+public class FriendStatusChangeEvent extends Event {
```

Comment:
Ideally, this class is defined by `Community`. Why does PGM need this?

Reply:
PGM needs the event to be able to react to state changes (eg: your tablist view re-rendering after you add someone as friend), the event is only launched externally tho. It's a similar mechanism to player prefixes/suffixes, if you have a change in state you should launch the event and pgm will re-render those names in tablist of make any updates that are needed to reflect the new prefix/suffix